src/interactions/panZoomInteraction.ts

```diff
@@ -271,9 +329,21 @@ export module Interactions {
      */
     public addYScale(yScale: QuantitativeScale<any>) {
       this._yScales.add(yScale);
+      this._ensureDefaultDomainExtents(yScale);
       return this;
     }
 
+    private _ensureDefaultDomainExtents(scale: QuantitativeScale<any>) {
+      if (this._minDomainExtents.get(scale) == null) {
+        this._minDomainExtents.set(scale, 0);
+      }
+      if (this._maxDomainExtents.get(scale) == null) {
+        var maxDateValue = 8640000000000000;
+        var defaultMaxDomainExtent = scale instanceof Scales.Time ? maxDateValue : Infinity;
```

Comment:
Same issue here -- the `instanceof` suggests that the `Scale`s should be implementing some of this functionality.